docs/topics/secured-mission-intro.adoc

```diff
@@ -31,28 +35,28 @@ a|
 The configured **clients accept any callback url**.
 
 a|
-To avoid having a custom configuration per runtime we avoid the verification of the callback as mandated by the OAuth2 specification.
+This example does not perform verification of the callback as mandated by the OAuth2 specification. A default configuration suitable for all runtimes was used.
 
 a|
-An application-specific callback URL should be provided with a valid domain name.
+In production, an application-specific (custom) callback URL with a valid domain name should be provided.
 
 a|
-**Clients do not require SSL/TLS and the secured applications are not exposed over HTTPS**.
+**Clients do not require SSL/TLS authentication and the secured applications are not exposed over HTTPS**.
 
 a|
-The boosters are simplified by not requiring certificates generated for each runtime.
+The boosters are simplified and don't require certificates to be generated for each runtime.
```

Comment:
Please expand the contraction here:

> don't require

to:

> do not require

Both the IBM Style Guide and the [Red Hat Style Guide](http://stylepedia.net/#sect-Red_Hat_Technical_Publications-Writing_Style_Guide-Grammar-Contractions_and_Abbreviations) discourage the use of cotractions.